leaderelection/src/test/java/io/scalecube/examples/LeaderElectionExample.java

```diff
@@ -0,0 +1,116 @@
+package io.scalecube.examples;
+
+import io.scalecube.cluster.Cluster;
+import io.scalecube.cluster.ICluster;
+import io.scalecube.leaderelection.IStateListener;
+import io.scalecube.leaderelection.LeaderElection;
+import io.scalecube.leaderelection.RaftLeaderElection;
+import io.scalecube.leaderelection.StateType;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+/**
+ * Created by ronenn on 9/12/2016.
+ */
+
+/*
+ * to run the example use the following commands: start -> this command will initiate seed node join <index> -> this
+ * command will join a node (seed) from the list list -> will list the current cluster state and leaders selected kill
+ * <index> -> will shutdown a node
+ * 
+ */
+public class LeaderElectionExample {
+
+  public static List<ICluster> nodes = new ArrayList<>();
+  public static Map<ICluster, LeaderElection> leaders = new HashMap<>();
+
+  public static void main(String[] args) {
+
+    System.out.println("to run the example use the following commands: "

```

Comment:
this is manual testing example very useful for manual testing i will add simple api based example